packages/client/src/utils/index.ts

```diff
@@ -126,18 +126,22 @@ export async function getVercelIgnore(
   prebuilt?: boolean,
   rootDirectory?: string
 ): Promise<{ ig: Ignore; ignores: string[] }> {
-  let ignores: string[] = [];
-
-  const outputDir = posix.join(rootDirectory || '', '.output');
+  const ig = ignore();
```

Comment:
Whats the difference between the ignore object here vs the ignore array?

Reply:
I'm not sure entirely but I didn't change anything related to the response here.